### PR TITLE
docs(readme): New logo & Readme(s) revamp

### PR DESCRIPTION
**Pull request by mateobelanger**
This pull request includes updates to configuration files and documentation improvements. The changes primarily focus on updating the `README.md` files for better clarity and adding new badges and examples. 

**For now the rig & playgrounds logos are added as files to the repo. If you agree with the style changes I'll move the files to the rig-site repo and switch to hyperlinks**.

### Documentation Updates:

* [`README.md`](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L3-R35): Updated logo sources, added new badges, improved wording, and added new sections for getting started and integrations. [[1]](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L3-R35) [[2]](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L45-R58) [[3]](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R82-R100)
* [`rig-lancedb/README.md`](diffhunk://#diff-208d8a0594e9a0d0ad10f99c1f98c92e58ca6e7b46a1a37a629a01595a4cbf37L1-R32): Updated the introduction with logos and added usage instructions.

close https://github.com/0xPlaygrounds/KnowledgeBase/issues/30

### CODE REVIEW

**Comment by 0xMochan** [comment]
image
interesting, i'm not sure about "By" not being aligned w/ "Playgrounds"

**Comment by mateobelanger** [comment]
> image
> interesting, i'm not sure about "By" not being aligned w/ "Playgrounds". Otherwise, it's very nice!
@0xMochan 
Thanks for the feedback! Yeah bizarre quirk of svg because in my editor it's aligned. Will probably have to use png/jpeg.

**Review by cvauclair — CHANGES_REQUESTED** [review]
Nice work! But there are a few things to change. Also, the README example is outdated and should be: 
```rust
use rig::{completion::Prompt, providers::openai};

#[tokio::main]
async fn main() {
    // Create OpenAI client and model
    // This requires the `OPENAI_API_KEY` environment variable to be set.
    let openai_client = openai::Client::from_env();

    let gpt4 = openai_client.agent("gpt-4").build();

    // Prompt the model and print its response
    let response = gpt4
        .prompt("Who are you?")
        .await
        .expect("Failed to prompt GPT-4");

    println!("GPT-4: {response}");
}
```
(change `openai_client.model` to `openai_client.agent`)

File: README.md [hunk 1]
```diff
@@ -1,32 +1,38 @@
...
 <a href=""><img src="https://img.shields.io/badge/built_with-Rust-dca282.svg?logo=rust" /></a>
 &nbsp;
-<a href="https://twitter.com/Playgrounds0x"><img src="https://img.shields.io/badge/Twitter-Follow_Us-1d9bf0.svg?style=flat /"></a>
-&nbsp;
-<a href="https://github.com/0xPlaygrounds/rig?tab=MIT-1-ov-file"><img src="https://img.shields.io/badge/license-MIT-00bfff.svg?color=red" /></a>
+<a href="https://twitter.com/Playgrounds0x"><img src="https://img.shields.io/twitter/follow/Playgrounds0x"></a>
+<a href="https://docs.rs/rig-core/latest/rig/"><img src="https://img.shields.io/badge/📖documentation-docs.rs-dca282.svg" /></a>
+<br>
 </p>
 &nbsp;
```
Comment (0xMochan) [inline README.md:22]: image
This seems wrong / duplicates.., might want to clean this up

**Pull request merged**